docs/Options.md

```diff
@@ -137,6 +138,18 @@ Directory is always shown and truncated to the value of `SPACESHIP_DIR_TRUNC`. W
 | `SPACESHIP_DIR_TRUNC_REPO` | `true` | While in `git` repo, show only root directory and folders inside it |
 | `SPACESHIP_DIR_COLOR` | `cyan` | Color of directory section |
 
+### Write permission (`write_perm`)
+
+Display a padlock if current directory is write-protected or if current user has not rights to write into current folder.
+
+| Variable | Default | Meaning |
+| :------- | :-----: | ------- |
+| `SPACESHIP_WRITE_PERM_SHOW` | `true` | Show section (`false` to hide) |
+| `SPACESHIP_WRITE_PERM_PREFIX` | `` | Prefix before section |
+| `SPACESHIP_WRITE_PERM_SUFFIX` | `·` | Suffix after section |
+| `SPACESHIP_WRITE_PERM_COLOR` | `red` | Color of the icon |
+| `SPACESHIP_WRITE_PERM_ICON` | `` | Icon displayed |
```

Comment:
What does this icon look like? In my terminal, that doesn't look like a padlock, but 🔒 does ([link](https://emojipedia.org/lock/)).

Also, I think this should be `SYMBOL`, not `ICON` (for consistency with the rest of spaceship).

Reply:
Apparently the currently used icon is from [FontAwesome](https://fontawesome.com/icons/lock?style=solid) and I didn't even notice this because I have FontAwesome installed, but I agree that the default icon should be from Unicode standard.

Reply:
The lock symbol from Powerline font actually looks not bad too, since other sections use Powerline symbols already maybe it can be a default?

![image](https://user-images.githubusercontent.com/1177900/35197651-878e5614-fee2-11e7-9e23-364086b11061.png)

Reply:
I would still prefer default unicode symbols when suitable ones exist. Many users forget to install powerline fonts. Some things are fairly obvious (e.g. ` master`), but some things aren't so clear 
(e.g. `/etc ` vs. `/etc 🔒`)

Reply:
agreed 👍

Reply:
We already require powerline font, so why not? Powerline lock icon looks more _native_ that Unicode one.

Reply:
> Powerline lock icon looks more _native_ that Unicode one.

I agree, but I think the argument was that it's you and me who installed Powerline fonts, but others might decided to not install it. So far it was mostly needed for git branch icon, and it is possible that many users decided that they are fine not installing Powerline font and seeing ` master` instead of ` master`. Such users will now start seeing `/etc `.

I'll let @nfischer continue, I'm fine with any icon as default 😛